README.md

```diff
@@ -161,15 +195,34 @@ You can work with KotlinDL interactively in Jupyter Notebook with the Kotlin ker
 For more details on installing Jupyter Notebook and adding the Kotlin kernel, check out the [Quick Start Guide](docs/quick_start_guide.md).
 
 ### Working with KotlinDL in Android projects
+
 KotlinDL supports an inference of ONNX models on the Android platform.
 To use KotlinDL in your Android project, add the following dependency to your build.gradle file:
+```groovy
+// build.gradle
+implementation 'org.jetbrains.kotlinx:kotlin-deeplearning-onnx:[KOTLIN-DL-VERSION]'
+```
 ```kotlin
-dependencies {
-    implementation ("org.jetbrains.kotlinx:kotlin-deeplearning-onnx:[KOTLIN-DL-VERSION]")
-}
+// build.gradle.kts
+implementation ("org.jetbrains.kotlinx:kotlin-deeplearning-onnx:[KOTLIN-DL-VERSION]")
 ```
 For more details, please refer to the [Quick Start Guide](docs/quick_start_guide.md#working-with-kotlin-dl-in-android-studio).
 
+## KotlinDL, ONNX Runtime, Android, and JDK versions
+
+This table shows the mapping between KotlinDL, TensorFlow, ONNX Runtime, Compile SDK for Android and minimum supported Java versions.
+
+| KotlinDL Version | ONNX Runtime Version | Minimum Java Version | TensorFlow Version | Android: Compile SDK Version |
```

Comment:
Perhaps, java version should be the first column?

Reply:
KotlinDL version in the first column is better for navigation, and it looks like a primary key to the row in the table instead of the rarely changed min Java version